packages/terra-consumer-icon/tests/jest/Icon.test.jsx

```diff
@@ -126,13 +126,13 @@ describe('Icon', () => {
     });
 
     describe('focusable prop', () => {
-      it('should have default focusable={false}', () => {
+      it('should have default focusable="false"', () => {
         const wrapper = shallow(<IconColorIncentives />);
-        expect(wrapper.prop('focusable')).toEqual(false);
+        expect(wrapper.prop('focusable')).toEqual('false');
       });
-      it('should have focusable={true}', () => {
-        const wrapper = shallow(<IconColorIncentives focusable />);
-        expect(wrapper.prop('focusable')).toEqual(true);
+      it('should have focusable="true"', () => {
+        const wrapper = shallow(<IconColorIncentives focusable="true" />);
```

Comment:
Yeah, this comes with react 16
see: [terra-core react 16 uplift](https://github.com/cerner/terra-core/pull/1194)
>The focusable attribute on SVG nodes takes a string as opposed to a boolean for it's value.

(unfortunate, because it seems like a dumb API)